easybuild/cray_external_modules_metadata.cfg

```diff
@@ -326,12 +376,20 @@ name = CUDA
 version = 9.1.85_3.18-6.0.7.0_5.1__g2eb7c52
 prefix = CRAY_CUDATOOLKIT_DIR
 
-
 [cudatoolkit/10.0.130_3.22-7.0.0.0_5.1__gdfb4ce5]
 name = CUDA
 version = 10.0.130_3.22-7.0.0.0_5.1__gdfb4ce5
 prefix = CRAY_CUDATOOLKIT_DIR
 
+[cudatoolkit/10.0.130_3.22-7.0.0.1_3.3__gdfb4ce5]
```

Comment:
⚠️  2 modulefiles found: 
```
-rw-r--r-- 1 root root 2596 Jan 16 20:46 10.0.130_3.22-7.0.0.0_5.1__gdfb4ce5
-rw-r--r-- 1 root root 2596 Feb 21 22:43 10.0.130_3.22-7.0.0.1_3.3__gdfb4ce5
```
but only /opt/nvidia/cudatoolkit10/`10.0.130_3.22-7.0.0.0_5.1__gdfb4ce5` is installed

Reply:
@jgphpc, should I remove the `[cudatoolkit/10.0.130_3.22-7.0.0.1_3.3__gdfb4ce5]`?